lab4-new.java/src/StringSearch.java

```diff
@@ -8,7 +8,7 @@
 
 public class StringSearch {
 
-    public static ArrayList<String> search(String word, File file) {
+    public static ArrayList<String> search(String word, File file, Boolean ignoreCase) {
```

Comment:
Consider using the word "pattern" and not "word" for clarity.

Reply:
Fixed, changed it